### PR TITLE
Add loadable attribute for LinearMulticlassModelParameters

### LINKED ISSUE #3209: [1.0.0-preview] Bug? - Getting an exception while loading .ZIP model MultiClass-Classification trained with SdcaNonCalibrated
I am trying to migrate IRIS Classification sample in the repo [here](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/MulticlassClassification_Iris) to **v1.0.0-preview**.

While loading the model for prediction I am getting the below error at the statement 

                trainedModel = mlContext.Model.Load(stream, out var modelInputSchema);

```
System.InvalidOperationException
  HResult=0x80131509
  Message=Error during class instantiation
  Source=Microsoft.ML.Core
  StackTrace:
   at Microsoft.ML.Runtime.ComponentCatalog.LoadableClassInfo.CreateInstanceCore(Object[] ctorArgs)
   at Microsoft.ML.Runtime.ComponentCatalog.TryCreateInstance[TRes](IHostEnvironment env, Type signatureType, TRes& result, String name, String options, Object[] extra)
   at Microsoft.ML.Runtime.ComponentCatalog.TryCreateInstance[TRes,TSig](IHostEnvironment env, TRes& result, String name, String options, Object[] extra)
   at Microsoft.ML.ModelLoadContext.TryLoadModelCore[TRes,TSig](IHostEnvironment env, TRes& result, Object[] extra)
   at Microsoft.ML.ModelLoadContext.TryLoadModel[TRes,TSig](IHostEnvironment env, TRes& result, RepositoryReader rep, Entry ent, String dir, Object[] extra)
   at Microsoft.ML.ModelLoadContext.LoadModel[TRes,TSig](IHostEnvironment env, TRes& result, RepositoryReader rep, Entry ent, String dir, Object[] extra)
   at Microsoft.ML.ModelLoadContext.LoadModelOrNull[TRes,TSig](IHostEnvironment env, TRes& result, RepositoryReader rep, String dir, Object[] extra)
   at Microsoft.ML.ModelLoadContext.LoadModel[TRes,TSig](IHostEnvironment env, TRes& result, RepositoryReader rep, String dir, Object[] extra)
   at Microsoft.ML.ModelOperationsCatalog.Load(Stream stream, DataViewSchema& inputSchema)
   at MulticlassClassification_Iris.Program.TestSomePredictions(MLContext mlContext) in C:\GitRepos\machinelearning-samples-v1.0.0-Preview\samples\csharp\getting-started\MulticlassClassification_Iris\IrisClassification\IrisClassificationConsoleApp\Program.cs:line 99
   at MulticlassClassification_Iris.Program.Main(String[] args) in C:\GitRepos\machinelearning-samples-v1.0.0-Preview\samples\csharp\getting-started\MulticlassClassification_Iris\IrisClassification\IrisClassificationConsoleApp\Program.cs:line 36

Inner Exception 1:
TargetInvocationException: Exception has been thrown by the target of an invocation.

Inner Exception 2:
InvalidOperationException: Error during class instantiation

Inner Exception 3:
TargetInvocationException: Exception has been thrown by the target of an invocation.

Inner Exception 4:
InvalidOperationException: Error during class instantiation

Inner Exception 5:
TargetInvocationException: Exception has been thrown by the target of an invocation.

Inner Exception 6:
FormatException: Couldn't load model: 'TransformerChain\Transform_002\Transform_000\Model'

```
the dataset format is like this 

```
#Label	Sepal length	Sepal width	Petal length	Petal width
0	5.4	3.7	1.5	0.2
0	4.8	3.4	1.6	0.2
0	4.8	3.0	1.4	0.1
0	4.3	3.0	1.1	0.1
```

I have pushed the code to the Github repo  here https://github.com/dotnet/machinelearning-samples/tree/migration/1.1.0-preview/samples/csharp/getting-started/MulticlassClassification_Iris

I did not understand the reason for the error. Could anyone help

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
fixes #3209.

### CODE REVIEW

File: src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs [hunk 1]
```diff
@@ -31,6 +31,10 @@
     "Multiclass LR Executor",
     MaximumEntropyModelParameters.LoaderSignature)]
 
+[assembly: LoadableClass(typeof(LinearMulticlassModelParameters), null, typeof(SignatureLoadModel),
+    "Multiclass LR No Prob",
```
Comment (abgoswam) [inline src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs]: >No Prob [](start = 19, length = 7)

should we say  "no calibration" or "raw scores" or "logits" ?

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs]: I'm afraid we have limit in 26 chars for this string, and no calibration doesn't fit.
I'm fine with `Raw score`, but I would prefer to @wschin  comment on this one, since he was one who split them apart

Comment (wschin) [reply src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs]: Multiclass LR No Calib",
